user_docs/en/userGuide.t2t

```diff
@@ -2865,6 +2874,13 @@ Specifically, following issues can be solved by running this tool:
 
 ++ Reload plugins ++[ReloadPlugins]
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
+You can also find out whether the application that you focus has an associated app module.

```

Comment:
We are currently in a paragraph describing the "Reload plugins" menu item. This sentence does not apply to this menu item and is thus a bit confusing, because there is no way to know the current appModule using a menu command.

Maybe you can replace by something more neutral announcing the table:

```suggestion

The following NVDA key commands may also be useful:
```